### PR TITLE
TeaserCarousel darkmode compatibility

### PR DESCRIPTION
**Pull request by ovbm**
Looks fine in Styleguide. Needs to be tested on live / production.

### CODE REVIEW

File: src/components/TeaserCarousel/Carousel.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ export const Carousel = ({
     : article
     ? MAX_WIDTH / 2 // optimised to align with article column
     : TILE_MAX_WIDTH
+  const [colorScheme] = useColorContext()
+
+  const bgColor = bgColorOverride || colorScheme.getCSSColor('default')
+  const color = colorOverride || colorScheme.getCSSColor('text')
```
[the remark this answers is not in the document]
Comment (tpreusse) [reply src/components/TeaserCarousel/Carousel.js:38]: Similarly a `lightBg` in `Row.js` could not be calculate from a `colorScheme.getCSSColor('default')`.

Let's pair on this tomorrow.

**Comment by ovbm** [comment]
@tpreusse I think this is what we discussed it should be, but now tiles don't take on bgColor defined at Carousel. Will have to check if i can figure out why.

**Review by tpreusse — APPROVED** [review]
Publikator now support `outline = true` for default divider color and `imageDark`:
Screenshot 2020-11-10 at 18 11 21
Screenshot 2020-11-10 at 18 11 32
PR: https://github.com/orbiting/publikator-frontend/pull/240

Allowing this to be produced:
Screenshot 2020-11-10 at 18 13 25

It's all on staging now. Needs final testing.

**Comment by ovbm** [comment]
looking good!

**Pull request merged**